hysds/job_worker.py

```diff
@@ -1438,6 +1438,26 @@ def handler(signum, frame):
             "celery_hostname": run_job.request.hostname,
         }
 
+    # overwrite message if _alt_msg.txt was dumped
+    msg = list()
+    alt_msg_file = os.path.join(job_dir, "_alt_msg.txt")
+    if os.path.exists(alt_msg_file):
+        with open(alt_msg_file) as f:
+            msgs = f.readlines()
```

Comment:
in the future should we expect users to write a ton of lines to `_alt_msg.txt`? (like thousands of lines per job)
should we change this to an iterator instead?

Reply:
Its basically implementation specific. For `msgs`, it normally should only be 1 line, similar to how the `_alt_error.txt` is created. As we've seen in the Track Frame Evaluator, there could be more than 1 informational short message, but in that case it could go maybe as high as 3 lines.

Reply:
Good question. It is implementation specific but the best practice would be to use `_alt_msg.txt` for high-level, generic messages (probably no more than 5) and to use `_alt_msg_details.txt` for details on those generic messages.